service/userService/userService_test.go

```diff
@@ -237,6 +246,10 @@ func TestUserService_GetMyCourses(t *testing.T) {
 			return true
 		case "2":
 			return true
+		case "3":
```

Comment:
I am thinking that these switches become very long. Maybe we could use a `map[string]bool` instead and use a lookup. Might be a bit more concise.